bin/MessagesBundle_el.properties

```diff
@@ -115,6 +115,22 @@ Restart= Εκκίνηση
 Drawing= Σχεδίαση
 FidoCad= Επεκτάσεις FidoCadJ
 pcbline_width = Πλάτος γραμμής PCB:
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on it.